src/routes/components/components.json

```
@@ -2362,6 +2362,16 @@
 		"category": "User Interaction",
 		"stars": 30
 	},
+	{
+		"title": "svelte-bricks",
+		"url": "https://github.com/janosh/svelte-bricks",
+		"description": "Naive Svelte masonry component without column balancing (ragged columns at the bottom)",
+		"tags": ["components and libraries", "masonry layout", "media sites"],
```

```suggestion
		"tags": ["components and libraries", "layout and structure"],
```
`masonry layout` and `media sites` are not existing tags